### DISCUSSION TITLE
Can 2 node-red instance connect through HA websocket addon with 0.63.1?

### DISCUSSION THREAD

**Discussion by goldriver2** [zachowj/node-red-contrib-home-assistant-websocket#1345] [Q&A]
Hi

updated to 0.63.1 and lost my nodes

p.s. I see that yellow triangle if I go in the pallette managing just under the websocket node: (I tried removin and re-installing and that triangle is still there)

![image](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/7215561/8185a653-58b4-435a-9e14-ecde269cc195)

thanks for any help on this

Node.js v20.10.0
Node-RED v3.1.3
HomeAssistrant v2024.3
node-red-cnontrib-home-assistant-websocket v0.63.1
node-red companion v3.1.4

downgraded to 0.60.0 and for now it seems to work

**Comment by zachowj** [comment]
Are you sure you're running NR 3.1.3? The only breaking change I can think of from 0.60 to 0.63 now requires NR 3.1.1+.

**Comment by goldriver2** [comment]
Confirm I was on 3.1.3, even just update to 3.1.8, still same issue

**Reply by zachowj** [reply]
What do your Node-RED logs show on startup?
